src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModel.scala

```diff
@@ -15,18 +12,20 @@ trait Attributable extends Identifiable {
  */
 case class WorkspaceName(
                       namespace: String,
-                      name: String) extends Identifiable {
-  def path : String = "workspaces/" + namespace + "/" + name
+                      name: String) {
   override def toString = namespace + "/" + name // used in error messages
+  def path(entity: Entity): String = path+s"/entities/${entity.name}"
+  def path(methodConfiguration: MethodConfiguration): String = path+s"/methodConfigs/${methodConfiguration.namespace}/${methodConfiguration.name}"
+  def path: String = s"/workspaces/${namespace}/${name}"
```

Comment:
Instead of having three methods here, could ```MethodConfiguration``` and ```Entity``` instead have their own ```path``` methods that take a ```WorkspaceName```?